.vscode/tasks.json

```diff
@@ -35,7 +37,8 @@
 			"group": "none",
 			"presentation": {
 				"close": true
-			}
+			},
```

Comment:
Can you please explain this change?

Reply:
this is just hiding this additional step of checking the output when launching the task